### ISSUE TITLE
readux-saga takeEvery warning

### ISSUE THREAD

**Issue by nickpalmer** [outlandishideas/kasia#62]
```
import { takeEvery } from 'redux-saga' has been deprecated in favor of import { takeEvery } from 'redux-saga/effects'.
The latter will not work with yield*, as helper effects are wrapped automatically for you in fork effect.
Therefore yield takeEvery will return task descriptor to your saga and execute next lines of code. 
```

**Comment by nickpalmer** [comment]
I am running redux-saga 0.15.3

**Comment by sdgluck** [comment]
Is this v4.1 or v4.2 beta?

**Comment by nickpalmer** [comment]
I am seeing it on v4.2 beta.

**Comment by seantjohnson-dev** [comment]
Any word on this yet? I'm using kasia 4.2.0-beta.4 and redux-saga 0.15.6 and still getting this message.

**Comment by sdgluck** [comment]
Fixed in #71, up on `kasia@beta`.
